pipelines/controller/src/main/resources/templates/index.html

```diff
@@ -59,19 +78,25 @@
       xhr.open("GET", '/status', true);
       xhr.onreadystatechange = function() {
         if (this.readyState === XMLHttpRequest.DONE && this.status === 200) {
-          status = this.response;
-          console.log('Received status: ' + status);
+          var data = this.response;
+          var json = JSON.parse(data);
+          status = json.pipelineStatus;
+          stats = json.stats;
+
           if (status != origStatus) {
             if (status == "IDLE") {
               changeVisibility('pipelineRunning', false);
               changeVisibility('pipelineDone', true);
             } else {
               changeVisibility('pipelineSubmit', true);
+              showPipelineProgress(stats)
             }
-            setTimeout(function() {
+            setTimeout(function () {
```

Comment:
nit: remove the additional space after `function` if it is not needed.

Reply:
Done